### DISCUSSION TITLE
Small quality of life improvement

### DISCUSSION THREAD

**Discussion by migueldeicaza** [migueldeicaza/SwiftGodot#69] [General]
Hello folks,

A small quality of life improvement.

I now publish binaries of my builds on the peer module https://github.com/migueldeicaza/SwiftGodotBinary that can be added to SwiftPM and can spare you the pain of having to build the package from source.   Currently it only does macOS binaries, I still need to figure out the steps for iOS.

**Comment by MarcusDobler** [comment]
Hi, for a mobile project i need some simple plugins for godot 4. For example i want to save a picture in the camera roll of the users phone. i would pay for the job, the plugins are needed for ios and andoroid. you can also release them as open source when done.

**Comment by Andrew-Dunn** [comment]
Amazing, I wish Swift PM had better support for binary distribution in general
